### DISCUSSION TITLE
Use Preview as Input is missing?

### DISCUSSION THREAD

**Discussion by dayunbao** [Mikubill/sd-webui-controlnet#2152] [Q&A] [answered]
I don't see this option in the GUI anymore.  I just tried using the `dw_openpose_full` preprocessor with the `thiebaud_xl_openpose` model (I'm finally swtiching to SD XL), and none of the images used the pose I generated.  Am I missing something obvious?
![screenshot](https://github.com/Mikubill/sd-webui-controlnet/assets/11196050/0fe3e1f3-abef-42f1-87b3-94cd08bfdb17)

**Comment by dayunbao** [comment]
Oh, well I'm running out of memory:

> RuntimeError: unable to mmap 2502139104 bytes from file </home/ec2-user/stable-diffusion-webui/extensions/sd-webui-controlnet/models/thibaud_xl_openpose.safetensors>: Cannot allocate memory (12)

That might have something to do with it.  Is Use Preview as Input no longer needed?

**Reply by dayunbao** [reply]
It was a memory issue.

**Comment by huchenlei** [comment, marked as answer]
See https://github.com/Mikubill/sd-webui-controlnet/issues/2076

**Reply by dayunbao** [reply]
These days I only use my UI and call endpoints, so I totally missed that change.  I'll have to remove the "Use preview as input" checkbox from my UI and stop sending it in requests.  Thanks!
